security/rbac/scripts/enable-rbac-connect.sh

```diff
@@ -0,0 +1,152 @@
+# - Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Topic:_secrets (for Secret Registry)
+# - Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Group:secret-registry (for Secret Registry)
+##################################################
+
+# Get the Kafka cluster id
+get_cluster_id_kafka
+
+for CONNECT_TOPIC in connect-configs connect-offsets connect-statuses; do
+  echo -e "\n# Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Topic:${CONNECT_TOPIC}"
+  echo "confluent iam rolebinding create --principal User:$ADMIN_CONNECT --role ResourceOwner --resource Topic:${CONNECT_TOPIC} --kafka-cluster-id $KAFKA_CLUSTER_ID"
+  confluent iam rolebinding create --principal User:$ADMIN_CONNECT --role ResourceOwner --resource Topic:${CONNECT_TOPIC} --kafka-cluster-id $KAFKA_CLUSTER_ID
+done
+
+echo -e "\n# Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Group:connect-cluster"
+echo "confluent iam rolebinding create --principal User:$ADMIN_CONNECT --role ResourceOwner --resource Group:connect-cluster --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding create --principal User:$ADMIN_CONNECT --role ResourceOwner --resource Group:connect-cluster --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+echo -e "\n# Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Topic:_secrets (for Secret Registry)"
+echo "confluent iam rolebinding create --principal User:$ADMIN_CONNECT --role ResourceOwner --resource Topic:_secrets --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding create --principal User:$ADMIN_CONNECT --role ResourceOwner --resource Topic:_secrets --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+echo -e "\n# Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Group:secret-registry (for Secret Registry)"
+echo "confluent iam rolebinding create --principal User:$ADMIN_CONNECT --role ResourceOwner --resource Group:secret-registry --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding create --principal User:$ADMIN_CONNECT --role ResourceOwner --resource Group:secret-registry --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+echo -e "\n# List the role bindings for User:$ADMIN_CONNECT for the Kafka cluster"
+echo "confluent iam rolebinding list --principal User:$ADMIN_CONNECT --kafka-cluster-id $KAFKA_CLUSTER_ID"
+confluent iam rolebinding list --principal User:$ADMIN_CONNECT --kafka-cluster-id $KAFKA_CLUSTER_ID
+
+echo -e "\n# Install kafka-connect-datagen"
+echo "confluent-hub install --no-prompt confluentinc/kafka-connect-datagen:latest"
+confluent-hub install --no-prompt confluentinc/kafka-connect-datagen:latest
+
+echo -e "\n# Bring up Connect"
+confluent local start connect
+
+
+##################################################
+# Connect client functions
+# - Grant the principal User:$ADMIN_CONNECT to the ResourceOwner role for Connector:$CONNECTOR_NAME
```

Comment:
This makes the service account for the Connect worker the ResourceOwner for the connector, which is a little strange.

Being the ResourceOwner for a connector allows someone to view its configuration, pause/resume it, create it, check its status, things like that. So that role should belong to a user, specifically, the one that issues the REST request to create the connector. Is there a user principal defined in this example?